lib/template-processing.js

```diff
@@ -77,15 +77,27 @@ const processPackageJson = (name, targetPath, { xdn, ...config } = {}) => {
     delete packageJson.scripts['rsf:link']
   }
   // remove react-storefront team from the deploy script, so it defaults to the user's own team:
-  packageJson.scripts.deploy = packageJson.scripts.deploy.replace('xdn deploy react-storefront', 'xdn deploy')
+  packageJson.scripts.deploy = packageJson.scripts.deploy.replace(
+    'xdn deploy react-storefront --environment=production',
+    'xdn deploy'
+  )
 
   _writeConfigJson(packageJsonPath, packageJson)
 }
 
+const copyResources = async (targetPath, { xdn }) => {
+  if (xdn) {
+    await fs.copy(path.join(targetPath, 'crs-resources'), targetPath)
```

Comment:
Where does crs-resources come from?

Reply:
It's the new directory from the [starter app PR](https://github.com/react-storefront-community/react-storefront-starter-app/pull/38) that I created to contain resources to copy when running `create-react-storefront`.